spec/features/collection_drafts/forms/add_another_button_spec.rb

```diff
@@ -24,7 +24,7 @@
 
     context 'when clicking the add another button' do
       before do
-        find('button', text: 'Add another Related URL').trigger('click')
+        find('button', text: 'Add another Related URL').click
       end
```

Comment:
Did the previous way not work?

Reply:
see the link I posted in the chat, it talks about this change being necessary.

Reply:
web browsers are very funny with how they handle events, at least if your an old hat like me and did it all by hand. I can see how APIs would not always get it right so switching methods I believe would work.

Reply:
there's a better explanation [here](https://about.gitlab.com/2017/12/19/moving-to-headless-chrome/), apparently `trigger` was a work around that Poltergeist provided (because the rendering was less accurate).